### DISCUSSION TITLE
can `angle` be replaced by using `glad` to use EGL 1.5 and GL ES 2.0 on Windows ?

### DISCUSSION THREAD

**Discussion by vtorri** [Dav1dde/glad#478] [Q&A] [answered]
I am porting to Windows the engine of a UNIX canvas.This canvas uses EGL 1.5 and GL ES 2.0. I have tried with `angle`, but i find it a bit overkill, espeecially if it is sufficient to load symbols.

So can `angle` be replaced by using `glad` to use EGL 1.5 and GL ES 2.0 on Windows ?

**Comment by Dav1dde** [comment, marked as answer]
No it cannot, you still need ANGLE when using glad. ANGLE is the driver, glad just connects your code with the driver.

**Reply by vtorri** [reply]
thank you
